### DISCUSSION TITLE
Bossbar displays only in Some Worlds

### DISCUSSION THREAD

**Discussion by VimaMT** [Zrips/CMI#5404] [Q&A]
Is there a way to create a bossbar that displays only in designated worlds? This would be useful if we want to give players a better visual when they are in a resource world.

**Comment by mrfloris** [comment]
That would be great, but as far as I know this isn't possible. It's a per player (or all players) option. 
Does vanilla Minecraft support bossbar msgs on a per world basis (or spigot's api?)

**Comment by VimaMT** [comment]
I can do this with the TAB plugin. I'm going to close the conversation for now.
